services/cannon/src/Cannon/App.hs

```diff
@@ -138,7 +137,7 @@ rejectOnError p x = do
         NotSupported              -> rejectRequest p (f "protocol not supported" "N/A")
         MalformedRequest _ m      -> rejectRequest p (f "malformed-request" (Text.pack m))
         OtherHandshakeException m -> rejectRequest p (f "other-error" (Text.pack m))
-        _                         -> throwM x
+        _                         -> pure ()
```

Comment:
So we now just ignore other errors instead of rethrowing the `HandshakeException`?

Reply:
no, this is not a behavioral change; the `throwM` is in the line below.

Reply:
Ah right; missed it 👍